apps/web/components/IncreasePower.tsx

```diff
@@ -156,7 +182,7 @@ export const IncreasePower = ({
   const { data: allowance } = useContractRead({
     address: registerToken,
     abi: abiWithErrors2<typeof erc20ABI>(erc20ABI),
-    args: [connectedAccount, communityAddress], // [ owner,  spender address ]
+    args: [accountAddress ?? "0x", communityAddress], // [ owner,  spender address ]
```

Comment:
you can disable the contract read until condition met:
https://1.x.wagmi.sh/react/hooks/useContractRead#enabled-optional

Reply:
I think I saw a couple other place with this problem in the PR, I mentioned some of them but I let you check if other I missed

Reply:
This ios great, i didn't know...
i will try to find and fix them all in my PR.
Still need to tell typescript that the variable is not undefined though.
![Screenshot from 2024-06-29 17-51-31](https://github.com/1Hive/gardens-v2/assets/54861577/b794f89d-6983-4f29-b2de-2cdb2f389c31)

Reply:
Yeah but at least not rpc quota used for nothing